### ISSUE TITLE
less-hacky build system on linux

### ISSUE THREAD

**Issue by classabbyamp** [GimelStudio/GimelStudio#167]
I'd potentially be interested in packaging this for void linux when it gets further along, and to make this easier, it would be nice if it was possible to use one of the standard ways (setup.cfg/setup.py/pyproject.toml) to install gimel studio. The current build.py script would not work for void for several reasons:

1. using a virtualenv
2. not using system python packages (i.e. using pip)
3. requiring brew to install oiio (this would ideally be done by void packaging oiio as a dependency)
    - this also has some side consequences in that script, like requiring exactly py3.9

**Comment by Correct-Syntax** [comment]
I would agree that we certainly do have a hacky script for building on Linux. We would definitely appreciate help in this area.

**Comment by classabbyamp** [comment]
I see that there's something in the works to switch to Qt. is this improvement still desired?

**Comment by iwoithe** [comment]
> I see that there's something in the works to switch to Qt (which would switch the build system to cmake)

This is correct. :+1:

But as it still has a *lot* of work to do...

> is this improvement still desired for the current iteration of the code?

Yes. :slightly_smiling_face:
